core/src/test/scala/unit/kafka/server/KafkaConfigTest.scala

```diff
@@ -761,6 +761,7 @@ class KafkaConfigTest {
   }
 
   @Test
+  @nowarn("cat=deprecation")
```

Comment:
please add a comment where ever we are adding this on which particular part of code is deprecated. It helps to ensure that this annotation is deleted when we remove the deprecated code.

Reply:
Updated everywhere. Thank you.